datamodel/src/main/scala/com/rasterfoundry/granary/Token.scala

```diff
@@ -0,0 +1,7 @@
+package com.rasterfoundry.granary.datamodel
+
+case class Token(
+    id: TokenId,
```

Comment:
Nice use of new types

Reply:
[This](https://tapir.softwaremill.com/en/latest/server/logic.html#providing-server-logic-in-parts-for-an-already-defined-endpoint) scared me 2/3 to death:

> Note that the partial logic functions should be fully typed, to help with inference. It might not be possible for the compiler to infer, which part of the input should be consumed.

so a newtype seemed like the easiest way to guarantee that there'd be no ambiguity. What if it picked the wrong string??